README.md

```diff
@@ -440,6 +440,135 @@ const subscriptionId = '<subscription_id>'; // player id
 await api.endLiveActivity('<app_id>', '<activity_id>', subscriptionId);
 ```
 
+## Users
+### Creating a OneSignal User
+```js
+const user = new OneSignal.User();
+
+const aliasLabel = '<alias_label>';
+const aliasId = '<alias_id>';
+const subscriptionToken = '<subscription_token>';
+
+user.identity = {
+    [aliasLabel]: aliasId,
+};
+
+user.subscriptions = [
+  {
+    type: "iOSPush",
+    token: subscriptionToken,
+  }
+];
+
+const createdUser = await api.createUser('<app_id>', user);
+assert(createdUser.identity!['onesignal_id'] != null);
+```
+
+### Getting a user by `onesignal_id`
+```js
+const fetchedUser = await api.fetchUser('<app_id>', "onesignal_id", createdUser.identity!['onesignal_id']);
```

Comment:
this one is confusing. the second argument appears to be passed in as is (i.e: "onesignal_id")

Reply:
It is passed as is. "onesignal_id" is a special case of an alias label. Restructured the code to make it less confusing.